server/app/index.js

```diff
@@ -4,6 +4,7 @@ const morgan = require('morgan');
 const compression = require('compression');
 const helmet = require('helmet');
 const sessionMiddleware = require('./sessionMiddlewarare');
+const sessiodIdMiddleware = require('./sessionIdMiddleware')
```

Comment:
is this supposed to be spelled "sessionIdMiddleware" instead of "sessiodIdMiddleware"?

Reply:
I guess it doesn't really matter how it is spelled, but just not sure if you intended this spelling